host/xtest/regression_1000.c

```diff
@@ -1603,6 +1616,30 @@ static void test_panic_ca_to_ta(ADBG_Case_t *c, const TEEC_UUID *uuid,
 	if (!ADBG_EXPECT(c, 0, counter))
 		goto bail2;
 
+	/*
+	 * Test TA panic from its open_session sequence
+	 */
+	TEEC_CloseSession(&cs[0]);
+	TEEC_CloseSession(&cs[1]);
+	TEEC_CloseSession(&cs[2]);
+
+	memset(&cs[0], 0, sizeof(cs[0]));
+	memset(&cs[1], 0, sizeof(cs[1]));
+	memset(&cs[2], 0, sizeof(cs[2]));
+
+	if (!ADBG_EXPECT_TEEC_SUCCESS(c,
+			xtest_teec_open_session(&cs[0], uuid, NULL, &ret_orig)))
+		return;
+
+	if (!ADBG_EXPECT_TEEC_RESULT(c, TEEC_ERROR_TARGET_DEAD,
+			sims_panic_in_open_session(&cs[1], uuid, &ret_orig)))
+		goto bail0;
+
+	exp_res = multi_instance ? TEEC_SUCCESS : TEEC_ERROR_TARGET_DEAD;
+	if (!ADBG_EXPECT_TEEC_RESULT(c, exp_res,
+				     sims_get_counter(&cs[0], &counter)))
+		goto bail1;
```

Comment:
The following piece of code should be place instead in order to avoid `Segmentation Fault` error in test execution. 
``` 
    ADBG_EXPECT_TEEC_RESULT(c, exp_res, sims_get_counter(&cs[0], &counter));
    goto bail0;
```